configure.ac

```diff
@@ -1511,20 +1511,28 @@ fi
 
 OPT_DARWINSSL=no
 AC_ARG_WITH(darwinssl,dnl
-AC_HELP_STRING([--with-darwinssl],[enable Apple OS native SSL/TLS])
-AC_HELP_STRING([--without-darwinssl], [disable Apple OS native SSL/TLS]),
+AC_HELP_STRING([--with-darwinssl],[enable use of Secure Transport])
+AC_HELP_STRING([--without-darwinssl], [disable use of Secure Transport]),
   OPT_DARWINSSL=$withval)
 
-AC_MSG_CHECKING([whether to enable Apple OS native SSL/TLS])
+AC_ARG_WITH(securetransport,dnl
+AC_HELP_STRING([--with-secure-transport],[enable sue of Secure Transport])
+AC_HELP_STRING([--without-secure-transport], [disable use of Secure Transport]),
+  OPT_DARWINSSL=$withval)
+
+AC_MSG_CHECKING([whether to enable Secure Transport])
 if test -z "$ssl_backends" -o "x$OPT_DARWINSSL" != xno; then
```

Comment:
Why don't we go all in, and change OPT_DARWINSSL to OPT_SECURETRANSPORT or something similar?